docs/content/integrations/pandas.mdx

```diff
@@ -3,6 +3,27 @@ title: Dagster with Pandas | Dagster
 description: The dagster-pandas library provides the ability to perform data validation, emit summary statistics, and enable reliable dataframe serialization/deserialization.
 ---
 
+<Note>
+  This page describes the <code>dagster-pandas</code> library which is used if you want to
+  perform data validation. If you are simply interested in using pandas with
+  dagster, we recommend starting with the{" "}
+  <a href="/tutorial/assets/defining-an-asset" target="new">
+    tutorial.
+  </a>{" "}
+  Dagster makes it easy to use pandas code to manipulate data and then store
+  that data in other systems such as{" "}
+  <a
+    href="/_apidocs/libraries/dagster-aws#dagster_aws.s3.s3_pickle_io_manager"
+    target="new"
+  >
+    files on s3
+  </a>{" "}
+  or{" "}
+  <a href="/integrations/snowflake/using-snowflake-with-dagster" target="new">
+    tables in Snowflake.
+  </a>{" "}
+</Note>
```

Comment:
```suggestion
  This page describes the <code>dagster-pandas</code> library, which is used for performing data validation. To simply use pandas with Dagster, start with the <a href="/tutorial/assets/defining-an-asset" target="new"> asset tutorial.</a>{" "}
  Dagster makes it easy to use pandas code to manipulate data and then store
  that data in other systems such as{" "}
  <a
    href="/_apidocs/libraries/dagster-aws#dagster_aws.s3.s3_pickle_io_manager"
    target="new"
  >
    files on Amazon S3
  </a>{" "}
  or{" "}
  <a href="/integrations/snowflake/using-snowflake-with-dagster" target="new">
    tables in Snowflake.
  </a>{" "}
</Note>
```

I edited this a bit for length and clarity, and also fixed some capitalization errors. Also specified that it's the asset tutorial, since that's what was linked. 

You'll probably have to run `make mdx-format` on this again